names/names.py

```diff
@@ -2,24 +2,24 @@
 
 start_time = time.time()
 
-f = open('names_1.txt', 'r')
+f = open('Sprint-Challenge--Data-Structures-Python/names/names_1.txt', 'r')
 names_1 = f.read().split("\n")  # List containing 10000 names
 f.close()
 
-f = open('names_2.txt', 'r')
+f = open('Sprint-Challenge--Data-Structures-Python/names/names_2.txt', 'r')
 names_2 = f.read().split("\n")  # List containing 10000 names
 f.close()
 
 duplicates = []
 for name_1 in names_1:
-    for name_2 in names_2:
-        if name_1 == name_2:
-            duplicates.append(name_1)
+    if name_1 in names_2:
```

Comment:
So this would still grade out to a time complexity of O(n^2); it's just not a strictly enforced O(n^2) like the initial setup